src/main/webapp/app/overview/student-questions/student-questions.scss

```diff
@@ -1,15 +1,108 @@
 @import 'src/main/webapp/content/scss/artemis-variables';
 
-.add-action {
-    color: $primary;
+$background-color: #353d47;
+
+.questions-container {
+    display: flex;
+    height: 100%;
+}
+
+.expanded-questions {
+    width: 30vw;
+
+    .draggable-left {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        min-width: 15px;
+    }
+
+    .card {
+        width: inherit;
+        min-width: 200px;
+
+        .card-header {
+            display: inline-flex;
+            justify-content: space-between;
+            align-items: center;
+            background-color: $background-color;
+            cursor: pointer;
+
+            .card-title {
+                display: flex;
+            }
```

Comment:
```suggestion
            }
            
            .row > .col-auto:last-child {
                display: flex;
                flex-direction: column;
                justify-content: center;
            }
```

Reply:
I implemented that as you said in the scss